lib/grape/exceptions/base.rb

```diff
@@ -7,12 +7,12 @@ class Base < StandardError
       BASE_ATTRIBUTES_KEY = 'grape.errors.attributes'
       FALLBACK_LOCALE = :en
 
-      attr_reader :status, :message, :headers
+      attr_reader :status, :headers
 
       def initialize(status: nil, message: nil, headers: nil, **_options)
         @status  = status
-        @message = message
         @headers = headers
+        super(message)
```

Comment:
Should be two lines above :)

Reply:
Does it matter? If you think it does, PR?

Reply:
#2304